make/ex_doc_wrapper.erl

```diff
@@ -0,0 +1,54 @@
+-module(ex_doc_wrapper).
+
+-export([main/1]).
+
+main(Args) ->
+    case os:find_executable("ex_doc") of
+        false ->  
+            case prompt() of
+                {ok, download} ->
+                    case download_otp() of
+                        error ->
+                            io:format("Unable to read ERL_TOP from environment?~n");
+                        ok ->
+                            main(Args)
+                    end;
+                {error, try_again} ->
+                    main(Args);
+                {error, do_not_download} ->
+                    io:format("Skipping download...~n")
+            end;
+
+        ExDocExe ->
+            Cmd = lists:append(["exec ", ExDocExe, " "],  lists:join(" ", Args)),
+            io:format("Running ~s~n", [Cmd]),
+            os:cmd(Cmd)
+    end.
+
+download_otp() ->
+    case os:getenv("ERL_TOP") of
+        false -> 
+            error;
+        ErlTop ->
+            Cmd = [ErlTop, "/otp_build download_ex_doc"],
+            io:format("Running ~s~n", [Cmd]),
+            Out = os:cmd(Cmd),
+            io:format("~s", [Out]),
+            ok
+    end.
+
+prompt() ->
+    case io:get_line("Do you want to download latest ex_doc from github? (y/n)? ") of
+        {error, _Description} -> 
+            io:format("Unable to get line?~n"),
+            {error, try_again};
+        eof ->
+            io:format("Got end of file?~n"),
+            {error, try_again};
```

Comment:
Not exactly sure what `{error, _}` would arise from but it seems safe to try again and let them `Ctrl-C`

I guess `eof` is just an empty line so we should try again.